atomic_reactor/plugins/pre_flatpak_create_dockerfile.py

```diff
@@ -39,13 +39,20 @@
 # /etc/os-release in the install root with the correct PLATFORM_ID
 # for our base package set. To make that work, we install system-release
 # into a *different* install root and copy /etc/os-release over.
+#
+# We also have to redo the addition of yum repos from the "pre_inject_yum_repo"
+# plugin after first removing any yum repos in the base image - we want
+# /only/ the yum repos from atomic_reactor, and nothing else.
 DOCKERFILE_TEMPLATE = '''FROM {base_image}
 
 LABEL name="{name}"
 LABEL com.redhat.component="{component}"
 LABEL version="{stream}"
 LABEL release="{version}"
 
+RUN rm -f {yum_repos_dir}*
+ADD {relative_repos_path}* '{yum_repos_dir}'
```

Comment:
Curious why only one of these gets quotes around it. Don't we want e.g. ``ADD '{relative_repos_path}'* '{yum_repos_dir}'`` etc?

Reply:
The second line is verbatim from `pre_inject_yum_repo.py`

```python
    # Insert the ADD line at the beginning of each stage
    df.add_lines(
        "ADD %s* '%s'" % (RELATIVE_REPOS_PATH, YUM_REPOS_DIR),
        all_stages=True, at_start=True, skip_scratch=True
    )
```

And then when I added the preceding line, I just did it the way that made sense to me - not pretending that `yum_repos_dir` was ever going to have spaces in it (adding the quotes that way doesn't help for anything but the "spaces" case - you'd need to use shlex.quote() or something.) But yes, the result is a inconsistent - I'd be happy to add or remove quotes for consistency.

Reply:
I see. Let's remove them then.